config/migrations/2024/20241028084700-expose-verkiezingen-to-ocmw/20241028084700-copy-verkiezingen.sparql

```diff
@@ -0,0 +1,26 @@
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+PREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>
+PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+PREFIX org: <http://www.w3.org/ns/org#>
+INSERT {
+  GRAPH ?ocmwG {
+    ?verkiezing mandaat:steltSamen ?gemeenteOrgInTijd.
```

Comment:
Why do we need this line? It is mentioned in the other queries, but I feel it is not really used and there is no easy way to differentiate this one from the ocmw one.